class.json-api-endpoints.php

```diff
@@ -1053,11 +1053,13 @@ function user_can_view_post( $post_id ) {
 	 * Returns author object.
 	 *
 	 * @param $author user ID, user row, WP_User object, comment row, post row
-	 * @param $show_email output the author's email address?
+	 * @param $show_email_and_ip output the author's email address and IP address?
```

Comment:
Now that you're writing this, you could fix the missing data type in these two params, object and bool respectively.